DuckDuckGo/FavoritesViewModel.swift

```diff
@@ -51,6 +51,10 @@ class FavoritesViewModel: ObservableObject {
     @Published private(set) var allFavorites: [FavoriteItem] = []
     @Published private(set) var isCollapsed: Bool = true
 
+    // In memory only so that when settings is dismissed we can show the prompt.
+    //  Missing icons will trigger the prompt from elsewhere too so we don't need to persist this.
+    var hasMissingIcons = false
```

Comment:
This doesn't seem to be modified from the outside
```suggestion
    private(set) var hasMissingIcons = false
```